invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -172,23 +173,44 @@ def _format_journal(journal, publication_date):
         journal_issn = journal.get("issn")
         journal_issue = journal.get("issue")
         journal_pages = journal.get("pages")
-        publication_date = f"({publication_date})" if publication_date else None
-        title = f"{journal_title}:"
-        issn = f"no. {journal_issn}" if journal_issn else None
-        issue = f"({journal_issue})" if journal_issue else None
-        pages = f"pp. {journal_pages}" if journal_pages else None
-        fields = [title, journal.get("volume"), publication_date, issn, pages, issue]
-        formatted = " ".join(filter(None, fields))
+        journal_volume = journal.get("volume")
+        publication_date_edtf = parse_edtf(publication_date).lower_strict() if publication_date else None
+        publication_date_formatted = f" ({publication_date_edtf.tm_year})" if publication_date_edtf else None
+
+        issn = f" no. {journal_issn}" if journal_issn else None
```

Comment:
Using "no." for ISSN is weird and makes it easy to confuse with issue. "ISSN: " would be a better label. Since ISSN isn't present in Zenodo it's not a change.